pytest/test_ssh_integration.py

```diff
@@ -312,3 +318,48 @@ def test_tags_async(sshd_manager, loop):
                     "sleep",
                     "1"
                 ]
+
+
+def tunnel_write_and_run(remote_write_fn, remote_cmd_fn):
```

Comment:
Doc string here, please. Also will be nice if you can provide a default `remote_write_fn` and `remote_cmd_fn` that an importer can re-use.